stream_alert_cli/test.py

```diff
@@ -907,6 +907,9 @@ def run_tests(options, context):
             options (namedtuple): CLI options (debug, processor, etc)
             context (namedtuple): A constructed aws context object
         """
+        os.environ['ALERT_PROCESSOR'] = '{}_{}'.format(
```

Comment:
Consider adding a comment here so it's clear why this is necessary

Reply:
Good idea, thanks